pkg/reporter/reporter.go

```diff
@@ -1,129 +0,0 @@
-package reporter
-
-import (
-	"errors"
-	"fmt"
-	"io"
-	"os"
-	"path"
-	"strings"
-
-	"github.com/golang/glog"
-	"github.com/onsi/ginkgo/v2/types"
-	"github.com/openshift-kni/k8sreporter"
-	"k8s.io/apimachinery/pkg/runtime"
-)
-
-var (
-	pathToPodExecLogs = "/tmp/pod_exec_logs.log"
-)
-
-func newReporter(
-	reportPath string,
-	namespacesToDump map[string]string,
-	apiScheme func(scheme *runtime.Scheme) error,
-	cRDs []k8sreporter.CRData) (*k8sreporter.KubernetesReporter, error) {
-	nsToDumpFilter := func(ns string) bool {
-		_, found := namespacesToDump[ns]
-
-		return found
-	}
-
-	if _, err := os.Stat(reportPath); os.IsNotExist(err) {
-		err := os.MkdirAll(reportPath, 0755)
-		if err != nil {
-			return nil, err
-		}
-	}
-
-	res, err := k8sreporter.New("", apiScheme, nsToDumpFilter, reportPath, cRDs...)
-
-	if err != nil {
-		return nil, err
-	}
-
-	return res, nil
-}
-
-// ReportIfFailed dumps requested cluster CRs if TC is failed to the given directory.
```

Comment:
nit: // ReportIfFailed dumps requested cluster CRs to the given directory if TC is failed.

Reply:
@achuzhoy This code is being deleted in this PR, so there are no changes to be made here